tests/test_usi.py

```diff
@@ -0,0 +1,34 @@
+from unittest.mock import Mock
+from unittest.mock import patch
+import numpy as np
+from matchms import Spectrum
+from matchms.importing.load_from_usi import load_from_usi
```

Comment:
```suggestion
from matchms.importing import load_from_usi
```

Reply:
Added the function to ``__init__.py`` in importing, so the last past can go.

Reply:
This seems to have broken something in the testing pipeline?

Reply:
I corrected the tests, mock.patch should use requests directly. I also applied a lint fix and shortened the import as suggested by Florian.

Reply:
Thanks!